packages/core-api/src/repositories/transactions.ts

```diff
@@ -248,6 +249,7 @@ export class TransactionsRepository extends Repository implements IRepository {
      * @return {Object}
      */
     public async getFeeStatistics(): Promise<any> {
+        const minFeeBroadcast = localConfig.get("dynamicFees.minFeeBroadcast");
```

Comment:
This should be resolved from the container, those config objects are only for plugin internal use.